blackbox/docs/appendices/release-notes/unreleased.rst

```diff
@@ -56,6 +56,8 @@ Breaking Changes
 Changes
 =======
 
+- Updated admin-ui with new help section and improved table/column view
```

Comment:
```suggestion
- Updated admin-ui with new help section and improved table/column view.
```

Reply:
I think we should highlight this fix a bit more https://github.com/crate/crate-admin/blob/master/CHANGES.rst#fixes.
I'd suggest to add an additionally `Fixes` entry e.g.:
`Fixed an issue in the admin-ui to no longer display all columns as being generated columns in the table/column view section.`

Then I'd phrase the changes entry like:
`Improved the help section of the admin-ui and added Spanish translations.`